graph/patterns/default-properties.md

```diff
@@ -0,0 +1,201 @@
+# Default properties
+
+Microsoft Graph API Design Pattern
+
+*The default properties pattern allows API producers to omit specific properties from the response unless they are explicitly requested using `$select`. The pattern also uses an instance annotation to inform callers that other properties are also available. The same annotation is also use to encourage callers to use `$select`.*
```

Comment:
```suggestion
*The default properties pattern allows API producers to omit specific properties from the response unless they are explicitly requested using `$select`. The pattern also uses an instance annotation to inform callers that other properties are also available. The same annotation is also use to encourage callers to use `$select`.*
```
can we move these two sentences to the Solution section?

Reply:
Sorry - are you asking to add these to the Solution section as well?  If I move them, there'll be nothing here :)

I can't actually see a difference with your suggested change, but did notice I need to change use -> used.

Reply:
I'm asking to move it to the Solutions section. I shouldn't have used the Suggestions feature at all; it should have just been a comment.

Reply:
Triage - move to the solutions section